### DISCUSSION TITLE
Madara Community Call #14

### DISCUSSION THREAD

**Discussion by tdelabro** [keep-starknet-strange/madara#1219] [Announcements]
## Agenda
- Madara tracks at Istanbul hacakton (@tdelabro)
- shared sequencing (@apoorvsadana)
- encrypted mempool (Radius team)

## Invitations
<a href="https://starkware-co.zoom.us/j/81102460303"><img src="https://img.shields.io/badge/Join%20on-Zoom-blue?logo=zoom" height="75" /></a>
<a href="https://calendar.google.com/calendar/u/0/r/eventedit/cTJzdXJwbmNiajNwM3A1b29tZDc0Z3Jya2sgdGltb3RoZWVAc3Rhcmt3YXJlLmNv"><img src="https://ssl.gstatic.com/calendar/images/dynamiclogo_2020q4/calendar_31_2x.png" height="75" /></a>

## Date & Time
- When: [Friday, 27 Oct 2023 at 4 PM CEST](https://www.timeanddate.com/worldclock/converter.html?iso=20231027T140000&p1=195&p2=179&p3=1440&p4=176&p5=240)

> "Wherever There Is Light — There Are Always Shadows."

**Comment by 0-knight** [comment]
Radius would love to share the result and progress about the proposal of encrypted mempool.
